packages/manager/apps/dedicated/client/app/otrs/index.js

```diff
@@ -0,0 +1,49 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'ovhManagerOtrsLazyloading';
+
+angular.module(moduleName, ['oc.lazyLoad', 'ui.router']).config(
+  /* @ngInject */ ($stateProvider, $urlRouterProvider, coreConfigProvider) => {
+    const region = coreConfigProvider.getRegion();
+    if (region === 'US') {
+      $stateProvider.state('app.otrs.**', {
+        url: '/ticket',
+        lazyLoad: ($transition$) => {
+          const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+          return import(
+            /* webpackChunkName: "otrs" */ './otrs.module'
+          ).then((mod) => $ocLazyLoad.inject(mod.default || mod));
+        },
+      });
+      $urlRouterProvider.rule(($injector, $location) => {
+        const path = $location.path();
+        if (path === '/support') {
+          return '/ticket';
+        }
+        if (/^\/support\/tickets/.test(path)) {
+          return path.replace(/^\/support\/tickets/, '/ticket');
+        }
+        return undefined;
+      });
+    } else {
+      $stateProvider.state('app.otrs', {
+        url: '',
+        template: '<data-ui-view></data-ui-view>',
+      });
+      $urlRouterProvider.rule(($injector, $location) => {
+        const path = $location.path();
+        if (path === '/ticket') {
+          return '/support';
+        }
+        if (/^\/ticket\//.test(path)) {
+          return path.replace(/^\/ticket\//, '/support/tickets/');
```

Comment:
Maybe in a (near) future we could uniformize urls as it is quite confusing to know when it is supposed to be `/ticket` or `/support/tickets`

Reply:
yes i suppose it was introduced for backward compatibility with old urls, i wonder if it's still necessary